src/Benchmarks/project.json

```diff
@@ -15,7 +15,8 @@
     "Microsoft.AspNet.Server.WebListener": "1.0.0-*",
     "Microsoft.AspNet.StaticFiles": "1.0.0-*",
     "Microsoft.Extensions.WebEncoders": "1.0.0-*",
-    "Newtonsoft.Json": "7.0.1"
+    "Newtonsoft.Json": "7.0.1",
+    "Jil": "2.13.0-*"
```

Comment:
Everything except Newtonsoft.Json uses wildcard... just sayin'

There are pros and cons of either approach; it saved me having to issue a PR when I last rev'd "dapper" ;p